docs/en/rules/Azure.MariaDB.FirewallRuleCount.md

```diff
@@ -0,0 +1,30 @@
+---
+severity: Awareness
+pillar: Security
+category: Network security and containment
+resource: Azure Database for MariaDB
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.MariaDB.FirewallRuleCount/
+---
+
+# Review Azure MariaDB server firewall rules
+
+## SYNOPSIS
+
+Determine if there is an excessive number of firewall rules.
+
+## DESCRIPTION
+
+Typically the number of firewall rules required is minimal, with management connectivity from on-premises and cloud application connectivity.
+
+Server-level firewall rules apply to all databases on the Azure Database for MariaDB server.
+
+## RECOMMENDATION
+
+Review Azure for MariaDB server firewall rules.
```

Comment:
I think we need a clearer message here about why the rule failed and what a customer should do.